src/voucherify/voucherify-connector.service.ts

```diff
@@ -44,19 +44,20 @@ export class VoucherifyConnectorService {
   }
 
   async validateStackableVouchersWithCTCart(
-    coupons: string[],
+    coupons: any,
```

Comment:
I suppose there could be coupons: Coupon[]. I checked this and no errors so far.